apps/studio/components/interfaces/Connect/ConnectionParameters.tsx

```diff
@@ -0,0 +1,95 @@
+import { ChevronRight, Copy, Check } from 'lucide-react'
+import { useState } from 'react'
+import {
+  Button,
+  cn,
+  Collapsible_Shadcn_,
+  CollapsibleContent_Shadcn_,
+  CollapsibleTrigger_Shadcn_,
+  Separator,
+} from 'ui'
+
+interface Parameter {
+  key: string
+  value: string
+  description?: string
+}
+
+interface ConnectionParametersProps {
+  parameters: Parameter[]
+}
+
+export const ConnectionParameters = ({ parameters }: ConnectionParametersProps) => {
+  const [isOpen, setIsOpen] = useState(false)
+  const [copiedMap, setCopiedMap] = useState<Record<string, boolean>>({})
+
+  const handleCopy = async (value: string, key: string) => {
```

Comment:
be useful to make a shared fn out of this

Reply:
we do have a helper here
https://github.com/supabase/supabase/blob/feat/connection-string-revamp/apps/studio/lib/helpers.ts#L149

could we switch it over?